src/openms/source/ANALYSIS/OPENSWATH/TargetedSpectraExtractor.cpp

```diff
@@ -289,6 +296,7 @@ namespace OpenMS
     ams.run(feat_map, output);
     // Remake the feature map replacing the peptide hits as features/sub-features
     feat_map_output.clear();
+    int unknown_feature_counter = 1;
```

Comment:
Is this used? Couldn't find it in the added code.

Reply:
Good catch.  It is removed.